app/scenes/Collection.js

```diff
@@ -75,18 +84,24 @@ class CollectionScene extends React.Component<Props> {
   }
 
   loadContent = async (id: string) => {
-    const collection = await this.props.collections.fetch(id);
-
-    if (collection) {
-      this.props.ui.setActiveCollection(collection);
-      this.collection = collection;
-
-      await this.props.documents.fetchPinned({
-        collectionId: id,
-      });
+    try {
+      const collection = await this.props.collections.fetch(id);
+
+      if (collection) {
+        this.props.ui.setActiveCollection(collection);
+        this.collection = collection;
+
+        await this.props.documents.fetchPinned({
+          collectionId: id,
+        });
+      }
+    } catch (error) {
+      if (error.name === "AuthorizationError") {
+        this.collection = null;
+      }
```

Comment:
catching specific error here, I wanted to do `instanceof AuthorizationError` but I didn't see an example anywhere else, so I went with duck typing instead

Reply:
You can use `instanceof`, it's in use for other error types